optimum/intel/utils/modeling_utils.py

```diff
@@ -267,6 +267,8 @@ def _infer_library_from_model_or_model_class(
 ):
     if model.__module__.startswith("open_clip"):
         library_name = "open_clip"
+    elif model.__module__.startswith("torch"):
+        library_name = "transformers"
```

Comment:
why this ?

Reply:
model.get_input_embeddings() returns instance of torch.nn.Embeddings class, if it is possible skip this check for such models, I think we can avoid this workaround

Reply:
added opportunity to propagate library_name from whole model to subcomponets